sql/hive/pom.xml

```diff
@@ -36,6 +36,12 @@
   </properties>
 
   <dependencies>
+    <!-- Added for sbt issue, expected to be evicted -->
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>2.10.3</version>
```

Comment:
So this is intentionally not matching the scala version since the plan is to this version evicted but still show up so sbt/ivy can make the decision to evict it.

I've nuked the ivy cache before and had this issue show up, but one of my system resolvers might be borked (I'll nuke the cache and disabled the system resolver and give it a shot tomorrow but that tends to take a long time).